code/game/machinery/computer/crew.dm

```diff
@@ -194,21 +198,25 @@ var/global/datum/crewmonitor/crewmonitor = new
 							area = format_text(player_area.name)
 							pos_x = pos.x
 							pos_y = pos.y
+							tracking_pos = ++_tracking.len
+							_tracking[tracking_pos] = H
 						else
 							area = null
 							pos_x = null
 							pos_y = null
+							tracking_pos = null
 
-						results[++results.len] = list(name, assignment, ijob, life_status, dam1, dam2, dam3, dam4, area, pos_x, pos_y, H.can_track(null))
+						results[++results.len] = list(name, assignment, ijob, life_status, dam1, dam2, dam3, dam4, area, pos_x, pos_y, H.can_track(null), tracking_pos)

```

Comment:
This is strictly to allow the AI to click a name if they have tracking beacon enabled and track them